app/views/projects/_selections.html.erb

```diff
@@ -1,29 +1,29 @@
 <div
   class="flex flex-row w-full items-center justify-between"
   data-controller="event"
-  data-event-day-not-allowed-value="<%= day_not_allowed %>"
+  data-event-aggregation-value="<%= aggregation %>"
 >
   <div class="flex flex-row gap-x-5 items-center justify-start">
     <div class="flex flex-col items-start justify-center">
       <%= label_tag 'events', 'Events', class: "text-sm font-medium mb-2" %>
       <%= select_tag "events",
-          raw(event_select_options(event_names, param_data, day_not_allowed)),
+          raw(event_select_options(event_names, param_data, aggregation)),
```

Comment:
Can we make sure to have a similar order of arguments for these methods?

maybe like: (`param_data`, `aggregation`, `...`) - since the first two are always passed along, this makes it more consistent (I know that it wasn't consistent before, but we can always improve)

Reply:
makes sense!